lte/gateway/c/core/oai/common/itti_free_defined_msg.c

```diff
@@ -222,6 +222,15 @@ void itti_free_msg_content(MessageDef* const message_p) {
       bdestroy_wrapper(
           &message_p->ittiMsg.s1ap_handover_request_ack.tgt_src_container);
       break;
+    case S1AP_HANDOVER_NOTIFY: {
+      e_rab_admitted_list_t e_rab_admitted_list = {0};
```

Comment:
Can you remove the `0` between the brackets because it initialises only the first field in the struct `e_rab_admitted_list_t`. It may lead to misinterpretations.